README.md

```diff
@@ -3,12 +3,13 @@
 [![npm version](https://badge.fury.io/js/angular2gridster.svg)](https://badge.fury.io/js/angular2gridster)
 
 > **Note**
-> This is a fork of [Angular2Gridster](https://github.com/swiety85/angular2gridster). It is updated to Angular 14 with some minor tweaks and updates added.
+> This is a fork of [Angular2Gridster](https://github.com/swiety85/angular2gridster). It is updated to Angular 15 with some minor tweaks and updates added.
```

Comment:
We are planning to keep the Angulat 14 tag kicking around in case anyone needs it. Can you please change the verbiage to Angular 14+

Reply:
Change is done as per comment.